test/e2e/runner/load.go

```diff
@@ -67,58 +65,47 @@ func Load(ctx context.Context, testnet *e2e.Testnet, multiplier int) error {
 }
 
 // loadGenerate generates jobs until the context is canceled
-func loadGenerate(ctx context.Context, chTx chan<- types.Tx, multiplier int) {
-	for i := 0; i < math.MaxInt64; i++ {
-		// We keep generating the same 1000 keys over and over, with different values.
-		// This gives a reasonable load without putting too much data in the app.
-		id := i % 1000
+func loadGenerate(ctx context.Context, txCh chan<- types.Tx, testnet *e2e.Testnet, id []byte) {
+	t := time.NewTimer(0)
+	<-t.C
+	for {
+		for i := 0; i < testnet.LoadTxBatchSize; i++ {
+			tx, err := payload.NewBytes(&payload.Payload{
+				Id:          id,
+				Size:        uint64(testnet.LoadTxSizeBytes),
+				Rate:        uint64(testnet.LoadTxBatchSize),
+				Connections: uint64(testnet.LoadTxConnections),
+			})
+			if err != nil {
+				panic(fmt.Sprintf("Failed to generate tx: %v", err))
+			}
 
-		bz := make([]byte, 1024) // 1kb hex-encoded
-		_, err := rand.Read(bz)
-		if err != nil {
-			panic(fmt.Sprintf("Failed to read random bytes: %v", err))
-		}
-		tx := types.Tx(fmt.Sprintf("load-%X=%x", id, bz))
+			select {
+			case txCh <- tx:
 
+			case <-ctx.Done():
+				close(txCh)
+				return
+			}
+		}
+		t.Reset(time.Second)
```

Comment:
Not sure about this, but this interval was configurable in `tm-load-test`. Do we want to do something similar here?

(I'm imagining we may want to increase it to test big TX bursts)

If my comment makes sense but the answer is "not needed now, can do later", I'm OK with that

Reply:
Yeah, "not needed now, can do later" was ultimately what I was thinking. The current e2e test only uses 1 second as the interval for sending load and the load testing done for v0.37 also only used 1 second. If we want to expand this capability in the future we certainly can but I'm not sure it's necessary to add until we're sure we're going to use it since it would ultimately be very simple to add.